cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala

```diff
@@ -186,8 +186,11 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
       title = "Listing indexes with filtering",
       text =
         """
-          |One way of filtering the output from `SHOW INDEXES` is to use the `BTREE` keyword to only list `BTREE` indexes.
-          |Another is to use the `WHERE` clause, for example to only show indexes not belonging to constraints.""".stripMargin,
+          |One way of filtering the output from `SHOW INDEXES` by index type is the type keywords,
+          |listed in the <<administration-indexes-syntax, syntax table>>.
+          |As an example, to show only B-tree indexes, use `SHOW BTREE INDEXES`.
+          |Another, more flexible way of filtering the output is to use the `WHERE` clause.
+          |For example to only show indexes not belonging to constraints.""".stripMargin,
```

Comment:
This sentence sounds like its not finished. Like its leading to an example that does not follow

Reply:
There is supposed to be an example after? (the query below?)

Reply:
Thats my point. The query below does not fit the text "only show indexes not belonging to constraints", right?

Reply:
- ```As an example, to show only B-tree indexes, use `SHOW BTREE INDEXES`.```
- `For example to only show indexes not belonging to constraints.`

`SHOW BTREE INDEXES WHERE uniqueness = 'NONUNIQUE'` shows all BTREE indexes not belonging to any constraints

Reply:
the example combines both filtering versions

Reply:
I am legit okay with any of the two as they both don't seem to reference the next query :-)

Reply:
not sure what you mean?

the query in the example is `SHOW BTREE INDEXES WHERE uniqueness = 'NONUNIQUE'` with
the WHERE clause is filtering on indexes without constraints and the BTREE keyword filters on type which is the 2 things the text describes?

Reply:
Ah okay, slight miscommunication, now I understand :-D So my problem is that the last sentence does not sound like its belonging to the rest of the paragraph but instead sounds like it's something new. And if you read it like that, then the query underneath does not fit.